test/integration/targets/docker_network/tasks/tests/options.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Registering network name
+  set_fact:
+    nname_1: "{{ name_prefix ~ '-network-1' }}"
+    nname_2: "{{ name_prefix ~ '-network-2' }}"
```

Comment:
I think you can delete `nname_2` here (and in the array in line 8).

Reply:
That was intentional, to match the format of the existing test packages. I recognize that it's not being used now, but it's not inconceivable that we'd need tests for multiple networks.

Reply:
The existing test packages always use as many names as they need (at least if I didn't forgot something myself :) ). If you don't want to remove it, fine for me as well.